3scaleAdapter/scripts/generate-config.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+# This script provides a way to generate the required files for an out-of-process adapter
+# Requires Istio source code to be on $GOPATH and will generate based on the locally checked
+# out version of Istio
+# Runs a diff of generated files against current configuration
+
+script_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+root_dir=$(dirname "${script_dir}")
+
+istio_src="${GOPATH-$HOME/go}/src/istio.io/istio"
+mixer_src="${istio_src}/mixer"
+target_dir="${mixer_src}/adapter/3scaleAdapter"
+
+
+# Test the existence of provided directory
+# Args: (path to directory)
+verify_dir_exists() {
+    if [[ ! -d "${1}" ]];
+    then
+        echo "Required directory not found in ${1}"
+        exit 1
+    fi
+}
+
+# Recursive copy of a given directory to a target directory
+# Args: (source directory, target directory)
+copy_files() {
+    cp -r "${1}" "${2}"
```

Comment:
a bit overkill to have a function for this, as we already have `copy_adapter_files`

Reply:
Ok will remove